packages/react-search-ui-views/package.json

```diff
@@ -69,6 +69,7 @@
   },
   "dependencies": {
     "autoprefixer": "^9.4.7",
+    "deep-equal": "^1.0.1",
```

Comment:
I added this for Facet filters ... we needed this to compare an applied filter to determine which value from a facet the current.

I.e., if the filter `{from: 1, to: 100}` is applied to the `visitors` field, in the list of ranges in the `visitors` facet, we need to compare the values of the visitors facets to determine which is applied, by doing a deep equal. 

Since facet values could range fitlers, geo filters, or values, the deep equal will be able to compare regardless of the type of filter / facet.

Reply:
TIL `This module is around 5 times faster than wrapping assert.deepEqual() in a try/catch.`